### ISSUE TITLE
Drag & Drop doesn't always work from Folders 

### ISSUE THREAD

**Issue by majkinetor** [CDrummond/cantata#964]
On majority of albums, but not all, reports _MPD reported the following error: Not found_. It works normally when clicking '+' button.

![cantata bug](https://cloud.githubusercontent.com/assets/85767/23136632/fb8a27e8-f79d-11e6-9ab9-c77987a7c030.gif)

Latest version.

**Comment by CDrummond** [comment]
Sorry, but I can't recreate this. Can you please try with Cantata's debug logging enabled?

1. From a cmd prompt enter:
setx CANTATA_DEBUG=1

2. Then launch cantata from the cmd window - c:\Program Files\Cantata\cantata

3. Recreate the issue. (Try to do as little as possible to create the issue - to save have to big a log file)

4. Cantata should have logged all its communication with MPD to C:\Users\USERNAME\AppData\Local\mpd\cantata\cache\cantata.log

5. Upload the log somewhere, and add a link here.

I dont use windows, so the commands above may not be 100% correct (never used *setx*)

**Comment by majkinetor** [comment]
Thx.

Found out what it is. If the folder ends with ALT+0160 symbol (simulates space) then this happens. When I remove it it doesn't. Shows as ? in the filename in the log:

https://gist.github.com/majkinetor/782a358e4653fbca0fc8aa8aab095587#file-cantata-log-L297

About log:

- [First I drag and drop 1 song](https://gist.github.com/majkinetor/782a358e4653fbca0fc8aa8aab095587#file-cantata-log-L2)  
here it does some strange 3 adds for some reason.
- [Then I add this same song with +](https://gist.github.com/majkinetor/782a358e4653fbca0fc8aa8aab095587#file-cantata-log-L297-L301) (I added dashed line between actions).
